pkg/time/comp.go

```diff
@@ -0,0 +1,8 @@
+package time
+
+import "time"
+
+// Between returns true if t is between start and end inclusive.
+func Between(t, start, end time.Time) bool {
```

Comment:
I don't know if it's worth checking for this, but should we verify that `start` is before `end`?  (I.e. this would return True if the end time supplied was earlier than the start time.)

Reply:
Oh, yeah, that's a good point. I'd probably just make it so the order doesn't matter:
```go
// Between returns true if t is between a and b inclusive.
func Between(t, a, b time.Time) bool {
	if b.Before(a) {
		a, b = b, a
	}
	return (a.Equal(t) || a.Before(t)) && (b.Equal(t) || b.After(t))
}
```

Reply:
PR #3